{"name": "Проверка ошибки при авторизации курьера с несуществующими данными", "status": "passed", "description": "Отправляем запрос на авторизацию в сервисе с несуществующими данными и проверяем ответ", "start": 1728126242497, "stop": 1728126242591, "uuid": "ed279308-c71f-4768-a7d6-1501be5d6308", "historyId": "7cbf374d46fe15da3f3640e2357167c3", "testCaseId": "7cbf374d46fe15da3f3640e2357167c3", "fullName": "test_login_courier.TestLoginCourier#test_courier_login_without_null_login_failed", "labels": [{"name": "suite", "value": "test_login_courier"}, {"name": "subSuite", "value": "TestLoginCourier"}, {"name": "host", "value": "Home-PC"}, {"name": "thread", "value": "15328-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_login_courier"}]}